src/test/java/BaseTest.java

```diff
@@ -1,11 +1,135 @@
+    public void closeBrowser(){
+        driver.quit();
+    }
+
+    protected static void clickSubmit() {
+        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitLogin.click();
+    }
+
+    protected static void enterPassword(String password) {
+        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
+        passwordInput.click();
+        passwordInput.clear();
+        passwordInput.sendKeys(password);
+    }
+
+    protected static void enterEmail(String email) {
+        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
+        emailInput.click();
+        emailInput.clear();
+        emailInput.sendKeys(email);
+
+    }
+
+    protected static void openLoginURL() {
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+    }
+
+    protected static void clickAvatarIcon(){
+        WebElement avatarIcon = driver.findElement(By.cssSelector("img.avatar"));
+        avatarIcon.click();
+    }
+
+    protected static String generateRandomName(){
+        return UUID.randomUUID().toString().replace("-","");
+    }
+
+    protected static void providePassword(String password){
+        WebElement currentPassword = driver.findElement(By.cssSelector("[name='current_password']"));
+        currentPassword.clear();
+        currentPassword.sendKeys(password);
+    }
+
+    protected static void provideProfileName(String name){
+        WebElement profileName = driver.findElement(By.cssSelector("[name='name']"));
+        profileName.clear();
+        profileName.sendKeys(name);
+    }
+
+    protected static void clickSaveButton(){
+        WebElement saveButton = driver.findElement(By.cssSelector("button.btn-submit"));
+
+    }
+
+
+    //HOMEWORK 17 BELOW
+    //
+    //
+    //
+    //
+    //
+
+    protected void verifySuccessMessageText() {
+        WebElement successNewPlaylist = driver.findElement(By.xpath(
+                "//div[contains(text(), 'new playlist17')]"));
+        Assert.assertTrue(successNewPlaylist.isDisplayed());
+    }
+
+    protected void selectSavePlaylist() {
+        WebElement savePlaylistButton = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/header/div[3]/div/section[2]/form/button"));
+        savePlaylistButton.click();
+    }
+
+    protected void enterNewPlaylistName(String newPlaylistName) {
+        WebElement createPlaylistField = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/header/div[3]/div/section[2]/form/input"));
+        createPlaylistField.click();
+        createPlaylistField.sendKeys(newPlaylistName);
+    }
+
+    protected void clickAddTo() {
+        WebElement addToButton = driver.findElement(By.cssSelector("[class='btn-add-to']"));
+        addToButton.click();
+    }
+
+    protected void clickFirstSong() {
+        WebElement firstSong = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/div/div/div[1]/table/tr"));
```

Comment:
the same here, first song WebElement can define as (//*`[@id='songResultsWrapper']//tr[contains(@class,'song-item')])[1]`